Svc/Framer/Framer.hpp

```diff
@@ -39,11 +39,6 @@ class Framer : public FramerComponentBase, public FramingProtocolInterface {
     Framer(const char* const compName /*!< The component name*/
     );
 
-    //! Initialize object Framer
-    //!
-    void init(const NATIVE_INT_TYPE instance = 0 /*!< The instance number*/
```

Comment:
Why was the init function deleted on `framer`? Also, why only on `framer` and not `deframer`?

Reply:
`init` functions are provided by the component base class and are, in older component implementations, improperly shadowed.  Thus, we are removing them.

Why not deframer?  Oversight.  We will remove it there too before merging.